src/app/_components/DontHaveAnAccount/DontHaveAnAccount.tsx

```diff
@@ -0,0 +1,16 @@
+import { Box, Typography } from "@mui/material";
+
+const DontHaveAnAccount = (): JSX.Element => {
+  return (
+    <Box display="flex" flexDirection="row" gap={1}>
+      <Typography id="dont-have-an-account" color={"#00000066"}>
```

Comment:
You can use `mui`'s `alpha` function to get a color with opacity (alpha) rather than using a 8 digit hex color code.

Reference:
https://github.com/mui/material-ui/blob/master/packages/mui-system/src/colorManipulator.js#L240

Example:
```
import {  alpha } from '@mui/material/styles';
// ...
<Typography id="dont-have-an-account" color={alpha('#000000', 0.66)}>
// or 
<Typography id="dont-have-an-account" color={alpha('black', 0.66)}>
```

Reply:
Resolved